src/autopas/LogicHandler.h

```diff
@@ -1085,7 +1085,10 @@ IterationMeasurements LogicHandler<Particle>::computeInteractions(Functor &funct
 
   functor.endTraversal(newton3);
   const auto [energyPsys, energyPkg, energyRam, energyTotal] = autoTuner.sampleEnergy();
+  long numFLOP = static_cast<long>(functor.getNumFLOPs());
+  double energyPerFLOP = static_cast<double>(energyTotal / 1e6) / static_cast<double>(numFLOP);
```

Comment:
```suggestion
  double energyPerFLOP = static_cast<double>(energyTotal) / numFLOP;
```
this should fix your tuning results with energyPerFLOP